MathlibTest/LibrarySearchRecDepth.lean

```diff
@@ -0,0 +1,19 @@
+import Mathlib
+
+-- this should not hit a recursion limit
+/--
+error: `exact?` could not close the goal. Try `apply?` to see partial suggestions.
+-/
+#guard_msgs in
+example (n : Nat) : n = 70 := by
+  exact?
+
+-- but it is the greatest literal not to do so
```

Comment:
Should this be a test? If this changes that's surely better?

Reply:
The point of the test is to record what the limit is

Reply:
Maybe add a comment along the lines of "if this fails please update this and the previous test appropriately"?

Reply:
Done